apps/payments/next/app/[locale]/[offeringId]/[interval]/checkout/[cartId]/success/page.tsx

```diff
@@ -155,6 +155,13 @@ export default async function CheckoutSuccess({
               width={91}
               height={24}
             />
+          ) : cart.paymentInfo.type === 'link' ? (
+            <Image
+              src={getCardIcon('link', l10n).img}
+              alt={l10n.getString('link-logo-alt-text', 'Link logo')}
+              width={70}
+              height={20}
```

Comment:
suggestion (non-blocking): Seems like there's an attempt to standardize around height=24. Could the change be made here as well, please?